R/MetAlyzer-seclass.R

```diff
@@ -24,21 +24,15 @@ MetAlyzer <- setClass(
   "MetAlyzer",
```

Comment:
Do we still need the custom MetAlyzer class? All the following functions could also directly be applied on a SE object.